docs/content/preview/develop/realworld-apps/ecommerce-app.md

```diff
@@ -151,7 +154,7 @@ The homepage is rendered by the `App` react component. The React route is the fo
 
 It uses the following REST API to query Express/NodeJS for all the products:
 
-```
+```sh
```

Comment:
I'd just use text for all these. `http` is an option too if there are actual methods like POST anywhere...
````suggestion
```text
````